src/shogun/kernel/GaussianKernel.cpp

```diff
@@ -121,3 +121,19 @@ float64_t GaussianKernel::distance(int32_t idx_a, int32_t idx_b) const
 {
 	return ShiftInvariantKernel::distance(idx_a, idx_b)/get_width();
 }
+
+SGMatrix<float64_t> 
+GaussianKernel::sample_spectral_density(int32_t dim_output, int32_t dim_input_space) const
```

Comment:
I thought we had discussed that this receives a rng as input?

Reply:
Indeed, I've hit a wall. Say, we're only expecting one type of PRNG right? The std::mt...  one?

Reply:
yes an rvalue reference of the prng of the RFF class that calls this method